docs/antora.yml

```
@@ -3,5 +3,6 @@ version: "nightly"
 title: Stackable Operator for Trino
 nav:
   - modules/getting_started/nav.adoc
+  - modules/usage_guide/nav.adoc
   - modules/ROOT/nav.adoc
```

```suggestion
  - modules/ROOT/nav.adoc
  - modules/usage_guide/nav.adoc
```

I suggest to have the concepts page first